update/ansible/playbook/tasks/update.yml

```diff
@@ -314,6 +322,29 @@
     - name: Check other services
       debug: var=update_result.stdout_lines
 
+    - name: Check that the SQLite grafana database exists
+      stat:
+        path: /srv/grafana/grafana.db
+      register: is_database_sqlite
+
+    - name: Migrate Grafana database from SQLite to Postgresql
+      include_role:
+        name: sqlite-to-postgres
+      when: is_database_sqlite.stat.exists
+      tags:
+        - skip_ansible_lint # '503 Tasks that run when changed should likely be handlers'.
+
+    - name: Fix grafana fields type
```

Comment:
should it be part of sqlite-to-postgres ansible book?

Reply:
Nope. Because we need it even for regular Grafana database